docs_src/design/adr/common-config-images/EdgeX 2.x Configuration bootstrapping flowchart.drawio

```diff
@@ -0,0 +1 @@
+<mxfile host="app.diagrams.net" modified="2022-11-29T21:09:02.803Z" agent="5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36" etag="WG-zQxjve1W6M5XXK2cf" version="20.6.0" type="device"><diagram id="C5RBs43oDa-KdzZeNtuy" name="Page-1">7Vxbd9o4EP41nLP7kBxfsLEfA7l1m3bTA22avuwRtgCnxqKygNBfv5ItX6UYQ8wlNC8BCVm25psZfTMjp6X3ps83GMwmn5AL/ZamuM8t/bKlaZpiavSD9aziHlXTjLhnjD2X92Udfe835J0K7517LgwLAwlCPvFmxU4HBQF0SKEPYIyWxWEj5BfvOgNjKHT0HeCLvQ+eSyZxr6V1sv5b6I0nyZ1V045/mYJkMF9JOAEuWua69KuW3sMIkfjb9LkHfSa9RC4PH1YP/t1P8+afL+Ev8LX7cfD521k82fUml6RLwDAgW089DGa9Xxer+XenrxuDW0L+8wZnZjz1AvhzLi++VrJKBAhdKk/eRJhM0BgFwL/KersYzQMXstsotJWNuUNoRjtV2vkECVlx5QBzgmjXhEx9/mvN9XE5hGiOHVgxTudqBvAY8vks5Wn+dO08aT+UmdtF/tLGAy4vhS0wpytcejcQTSHBKzoAQx8Qb1FUKMD1cpyOy2RPv3Dxb4CyLkBxh4BLp/oEMFU9er3p09V0h5h+G5NIbCaYUgl3g2HIPkYYTZmJeBSVMooZRkzgy4lHYH8GIjEuqeUX8RihgHCwVCqQ7tgHYcjhDQlGP1NTYqNTu1A2hnIBMYHPlcJPfjU5WqvEM/H2MjNqNbHUSc6g28rr8ZKaTkfA6zM6AeMxReORz1fXejiAZ8p5RytAmMxQ27z43PfIoyvNhqDRKKSPWsYzfYTtIbYEiB/pVnYAjCm0ePWdXX9uJM1HPl3UuHwutFZbWWNjutHZWDUsxbCLymEekXJUCSOnHF9D6neVPnQwu6Nyj9GCch8s+uIJmg7n4Xo/XPCszClfg6nnMwndQn8BiecAibcGvjcOaMOhoNO7S102vaUXjGnLzFqDSBnP2jv14kYBZb1til5clXhxqwEvvrxdzeb9/uOnj/6XH98+Xs9ue3aq+m/InEtm1aB9dzby/a8gStGlFxiDVW7AjJluKLHlJBjRihSgbSol+OMZG7VycZfvYQhITUN/46RLb5ck3j406bLfd2QpC1sb69gbb8l2J4mRky1Zfd2WzKduq0WlMrWSssRr4VflA9r1e/umLsUwis9idFT5LV8Ynyj6S+PN6vGvdllSG0nk+24QlROqNS2i8ehfDlrinNeGk3dgCP2i7AXGx1w8JYf+Bf9h6rluDDMMvd9gGM3HpM8NhE5udFvGpRSPSiUTNpM0k8fv0sony15wNYZd9DRnzXiaRKbprEppit0FB6l4ytGBmMXpoWDkjQs0QjLqr9793yK/+HMDibLj1iSBRFvCTOwGmEmlP8khfomC00/E2ftMxMmXoAmSTzl6alw934OcT5wSGoZdZuidc2N/eEippnYQ+vHskRz7oK3H3C8Z92CN11OP/ZQT6oXBaQ5lDX/eYmurWmbO2i5mM58t7F+qtziq9p2amQnVh84enZ4UhbaAwmACI4YRix7OQnZbDCOF+zX3MBU/K73SP0PW6UabkzLycEgi7Y6ZG/WWc0zVEwXsCaI5FsDzIzLHL3cS5yqyFElKhN7NR8vzcq1KUBGKBCmRlgjrHvIR5R6XAXteqhye75e66rNfmeIVPc+uFEgr+WlNFxUoLWnlFchsQIHkBF7cNf+YVEpVhmRtJsWs6eZzxQ3dbDaTsvvKlypWo99adbMuyJWccpPqpmVpRZSPqYRVJY33EtYGntxUSp7c0mSMW+bLmyhiya21fVC/rRa8dubE9+O3ax9YqGvSu61iWYZZ0J+2mURsTSWFq6SUM/UbZuFlUqZcOA4MGe0bUHsLRLt/40S+Yxelr5tS691rTUs13plYQVcTd7aWiqnWFlzMKB5BOKpdWq4fou2eBBerDXOCxibFS7WU/D8qmCvFkU9j3jMfBMI0NH/nYS/yMEOe+dwVD5NDaB7Ea+889bnrlKbZtl/CsvahgBemalvCVLFj2ll+VBVPjW55kJuZ/4nxL6tktaZ2BPyrc1CuddjoSZVkRORSqnuCaMfhUykTb+0nfFLFVMnpVz3KxtpRj6C4eJjDTVtssfsJptbHSLuxW9EwrWJk3VG23sMtwz4vBmlmFqY3v4vL3Z110E3hsAG4JlGu49gUmiZryaPk/Pr9PJy0eKnzBBlY+cTIUTh1kTL3oTOPiteIsmTlQxAm7awkUc5dlln1B1bCTq+bIpd9eMzPzEO2WytkkhU7+gRFw5I4W/EpZ49K5lVvVZ520VpXtIKq5GKpvKokKdW8qui7UpVk4rzBshe12ZvhTFNANEWEukdW0Ve88BwJOztt7EpWbsqhU/cKncTXitBRGZ+9w6eXXpIwOrI4WZOdcG0CP+lmL5YpDkGRtqc60kXVfV1Ab5rovAoK0Qn26Qp4xZ4V8FhGNjv9emIUpnwaT5UcptpvCklCI0/ANmS5oTfxLo2YrEnMYwmIM4mNI950Hug+ww83OhMQjE8wjVNm/NZOz4jTZvYvceIwLfvPQvrV/w==</diagram></mxfile>
```

Comment:
Best to check-in the uncompressed and formatted export for better version control.

Reply:
Why is this needed? 
It is here for when the image need to be updated, no one will review it.
This is the format that https://www.diagrams.net/ stores it in. It could be exported to XML, but that is just more unneeded work.

Reply:
The raw source is indeed helpful to do reviews in case someone modifies it. I remember @brian-intel has this discussion some time ago. I'm not insisting on doing so, it is just nice to have.

Reply:
The raw source is indeed helpful to do reviews in case someone modifies it. I remember @bnevis-i has this discussion some time ago. I'm not insisting on doing so, it is just nice to have.